src/Http/Controllers/Support/SearchController.php

```diff
@@ -26,7 +26,7 @@
 use Seat\Eveapi\Models\Mail\MailHeader;
 use Seat\Services\Search\Search;
 use Seat\Web\Http\Controllers\Controller;
-use Yajra\Datatables\Datatables;
+use Yajra\DataTables\DataTables;
 
 /**
  * Class SearchController.
```

Comment:
# Search

I had a lot of errors on this one while attempting to search things or sorting values.

## Character Results
- sorting on a column is not triggering the id-to-name resolver; which mean corporation stay unresolved

## Mail results
- sorting on a column is not triggering the id-to-name resolver
- sorting on date is not working properly (maybe a missing date attribute ?)
- sorting on title is not sorting properly
- sorting on content is throwing an exception

```
DataTables warning: table id=mail - Exception Message:
Trying to get property of non-object
```

- sorting on `to` column is throwing an exception

```
DataTables warning: table id=mail - Exception Message:

		SQLSTATE[42S22]: Column not found: 1054 Unknown column 'tocounts' in 'order clause' (SQL: select * from `mail_headers` where (LOWER(`mail_headers`.`timestamp`) LIKE %warlof% or LOWER(`mail_headers`.`subject`) LIKE %warlof% or exists (select * from `mail_bodies` where `mail_headers`.`mail_id` = `mail_bodies`.`mail_id` and LOWER(`mail_bodies`.`body`) LIKE %warlof%)) order by `timestamp` desc, `tocounts` asc limit 10 offset 0)
```
## Character Assets
- sorting on name is not sorting properly

## Character Skills
- sorting on a column is not calling id-to-name resolver